### PR TITLE
Add loading wheel to android

### LINKED ISSUE #44: Add loading animation while loading journeys on Android

### PR DESCRIPTION
**Pull request by billy-yoyo**
closes #44

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -74,14 +74,30 @@
         app:layout_constraintTop_toBottomOf="@id/arrival_station"
         android:onClick="buttonClick"/>
 
-    <view class="androidx.recyclerview.widget.RecyclerView"
-        android:id="@+id/journey_recycler"
+    <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (cameronmccormack) [inline app/src/main/res/layout/activity_main.xml]: These declarations of `xmlns:android` and `xmlns:tools` are redundant as already declared in the parent

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -74,14 +74,30 @@
...
+
+        <view class="androidx.recyclerview.widget.RecyclerView"
+            android:id="@+id/journey_recycler"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            >
+        </view>
+    </FrameLayout>
+
+
```
Comment (cameronmccormack) [inline app/src/main/res/layout/activity_main.xml]: too many blank lines?

Comment (billy-yoyo) [reply app/src/main/res/layout/activity_main.xml]: fixed

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (billy-yoyo) [reply app/src/main/res/layout/activity_main.xml]: removed

**Pull request merged**